src/components/Purpose/common/FieldProduct.tsx

```diff
@@ -1,27 +1,31 @@
 import * as React from "react";
-import {Select, Value} from "baseui/select";
+import {Select} from "baseui/select";
 import {codelist, ListName} from "../../../service/Codelist";
-import {FieldArray} from "formik";
-
-const FieldProduct = (props: { products: string[] }) => {
-  const {products} = props
-  const [value, setValue] = React.useState<Value>(products ? products.map(product => ({
-    id: product,
-    label: codelist.getShortname(ListName.SYSTEM, product)
-  })) : [])
+import {FieldArray, FormikProps} from "formik";
+import {ProcessFormValues} from "../../../constants";
+import {renderTagList} from "../../common/TagList";
+import {Block} from "baseui/block";
 
+const FieldProduct = (props: { formikBag: FormikProps<ProcessFormValues> }) => {
   return <FieldArray
     name='products'
-    render={arrayHelpers => <Select
-      multi
-      clearable
-      options={codelist.getParsedOptions(ListName.SYSTEM)}
-      onChange={({value}) => {
-        setValue(value)
-        arrayHelpers.form.setFieldValue('products', value.map(v => v.id))
-      }}
-      value={value}
-    />}
+    render={arrayHelpers => (
+      <>
+        <Block width={'100%'}>
```

Comment:
Litt pirkete, men du trenger ikke å skrive width={'100%'}, vi har for det meste skrevet det bare i en streng width="100"

Reply:
Det kommer automatisk fra IntelliJ :)